rmgpy/molecule/molecule.py

```diff
@@ -1621,4 +1621,41 @@ def toGroup(self):
         
         return group
 
+    def getAromaticSSSR(self):
+        """
+        Returns the smallest set of smallest aromatic rings
```

Comment:
also I realize you might not want other people to use this method once the `generateResonanceIsomers()` is finally done. Because then a more elegant way of getting smallest set of smallest aromatic ring is 
1. get a Species object from the target molecule
2. generateResonanceIsomers()
3. loop over all the molecules to get the aromatic isomers
4. examine these aromatic isomers to figure which rings are aromatic